BDArmory/Distribution/GameData/BDArmory/Localization/UI/en-us.cfg

```diff
@@ -686,6 +686,7 @@ Localization
         #LOC_BDArmory_EvasionTimeThreshold = Evasion Time Threshold	
         #LOC_BDArmory_EvasionMinRangeThreshold = Evasion Min Range Threshold
         #LOC_BDArmory_EvasionIgnoreMyTargetTargetingMe = Don't Evade My Target
+        #LOC_BDArmory_EvasionMissileKinematic = Evade Missiles Kinematically
```

Comment:
![Screenshot_20240108_223210](https://github.com/BrettRyland/BDArmory/assets/18440126/8d25e8ef-627d-4ccf-821b-cd8e037c7539)
The text is pretty small here. You might want to condense it a bit.